docker/moj.json

```diff
@@ -10,6 +10,7 @@
     {"type":"vcs", "no-api": true, "url":"https://github.com/ministryofjustice/intranet.git"}
   ],
   "require": {
+    "wpackagist-plugin/breadcrumb-navxt":"5.7.1",
```

Comment:
It seems like a sensible choice and I'd much rather use established plugins wherever possible.